src/raft.rs

```diff
@@ -439,12 +439,16 @@ impl<T: Storage> Raft<T> {
         };
         for p in peers {
             let pr = new_progress(1, r.max_inflight);
-            r.mut_prs().insert_voter(*p, pr);
+            if let Err(e) = r.mut_prs().insert_voter(*p, pr) {
+                panic!("{}", e);
```

Comment:
em, I can't see the difference between panic inside the `insert_voter` and here, they are all in the Raft module and outside user can't know it.

Reply:
Later we can go and make `new()` return a `Result` when it fails instead of having a panic. It will be a follow up ticket.

Reply:
@Hoverbear 

This function is not only used in the new function, other places also use it in depth, if we want to return error, we need to change a lot for many functions.